Makefile

```diff
@@ -45,13 +58,29 @@ vet:
 # build targets
 #
 
-bin/%:
-	$(Q)echo "Building $@..."; \
-	$(GO_BUILD) -o $@ ./$(subst bin/,cmd/,$@)
+$(BINARIES): bin/%:
+	$(Q)echo "Building $@..."
+	$(Q)(cd cmd/$(*) && $(GO_BUILD) -o $(abspath $@) .)
```

Comment:
nitpick: I've usually try to unconditionally and habitually use everywhere multiline shell commands prefixed with a single `$(Q)`... just to avoid mistakes where a failure (by something else than the last command) should cause make to fail, but it doesn't. I know it's irrelevant here and I'm fine with this as it is. Just noticed and decided to comment anyway...

Reply:
OK. I can update to follow your pattern here too. I don't have strong feelings about it.